cvr/src/main/java/org/cbioportal/cmo/pipelines/cvr/model/staging/CVRSvRecord.java

```diff
@@ -105,7 +109,61 @@ public CVRSvRecord(CVRSvVariant variant, String sampleId) {
         this.tumor_read_count = variant.getTumor_Read_Count();
         this.tumor_variant_count = variant.getTumor_Variant_Count();
         this.variant_status_name = variant.getVariant_Status_Name();
+
+        // CVR confirmed Site1_GENE/Gene1 is a NOT_NULL field
+        // Use Gene1 to test whether v1 or v2 schema
+        if (variant.getSite1_Gene() != null && !variant.getSite1_Gene().isEmpty()) {
+            this.site1_gene = variant.getSite1_Gene();
```

Comment:
Delete this.site1_gene and this.site2_gene initialisation above.

Reply:
Check for calls to setGene1 and setGene2 to see if site1_gene and site2_gene needs setting in that case.

Reply:
Avery looked it over and said these are only called once and only for the data_sv.txt not the cvr json file (so this is correct behaviour).